products/artifactregistry/terraform.yaml

```
@@ -24,6 +24,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           repository_id: "my-repository"
           description: "example docker repository"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "artifact_registry_repository_cmek"
+        min_version: 'beta'
+        primary_resource_id: "my-repo"
+        vars:
+          repository_id: "my-repository"
+          kms_key_name: "kms-key"
+        test_vars_overrides:
+          kms_key_name: 'BootstrapKMSKeyInLocation(t, "us-central1")'
```

```suggestion
          kms_key_name: 'BootstrapKMSKeyInLocation(t, "us-central1").CryptoKey.Name'
```

Give this a try! I think that's what we need to get tests passing.